dashboard/cron.py

```diff
@@ -247,6 +245,15 @@ def update_with_bq_access(self):
         # the earliest start date of all courses
         course_start_time = utils.find_earliest_start_datetime_of_courses()
 
+        # the earliest latest date of all resources_accessed
+        try:
+            latest_resource = ResourceAccess.objects.latest('access_time')
+            latest_resource_time = latest_resource.access_time
+        except ResourceAccess.DoesNotExist:
+            latest_resource_time = course_start_time
+
+        logger.info(f"Earliest Start: {course_start_time} Latest resource: {latest_resource_time}")
```

Comment:
I find this log message somewhat confusing. I think if it were me, I'd nest log messages in the `try/except` and log something different depending on what source you use (courses or resource access).

Reply:
Okay, that makes sense. I'll have 2 custom messages.